### DISCUSSION TITLE
Inverse Pole Figure - modifying indices

### DISCUSSION THREAD

**Discussion by liuqing72** [mtex-toolbox/mtex#1900] [Ask Anything] [answered]
Hi community,

I am working on Ni-based alloys with m-3m symmetry, The format indices of inverse pole figure is [001], [011],[-1 1 1]. Any codes I should add to change it to [001], [101]and [111]. please see attached images. How to switch from image 1 to image 2?

Thanks,
Liuqing

![Capture1](https://github.com/mtex-toolbox/mtex/assets/139285961/e430c8f3-e1b6-4b65-aa1b-34e60fa12f4a)

![Capture2](https://github.com/mtex-toolbox/mtex/assets/139285961/f2fd2a71-5416-4645-bda2-ee2b31ec4019)

**Comment by kilir** [comment, marked as answer]
Hi,
see here https://github.com/mtex-toolbox/mtex/discussions/1894#discussioncomment-6435184
Cheers,
Rüdiger

**Reply by liuqing72** [reply]
Dear friend,

That works perfect! Thank you so much!!!

Best,
Liuqing
